web/src/Setup.tsx

```diff
@@ -243,6 +243,9 @@ export default function Setup() {
                         Go back to the exporter configuration step to make
                         changes and click "Next" once you are satisfied.
                       </Box>
+                      <Box variant="p">
+                        If your labgrid configuration is not yet successful, you can correct it later.
+                      </Box>
```

Comment:
It's not the configuration that is successful or not, isn't it? It is the labgrid-exporter startup that is successful or not depending on the configuration.

I am having a hard time coming up with a better formulation right now. Maybe something like:

```suggestion
                      <Box variant="p">
                        If your labgrid configuration does not work right away you can always come back and change it later.
                      </Box>
```

Reply:
I has some different wordings and wasn't happy with them anyway. I'll take yours :+1: